customerRegistration/src/main/java/DAO/DBConnection.java

```diff
@@ -0,0 +1,38 @@
+package DAO;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DBConnection {
+
+	private String jdbcUrl = "jdbc:mysql://localhost:3306/hotwaxsystem?useSSL=false";
+	private String user ="root";
+	private String pass="123456";
```

Comment:
Please use property file to save database credentials.

Reply:
Done